app/src/components/page/Sidebar/utils.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import { DashboardOutlined, IdcardOutlined } from '@ant-design/icons';
+import { Dictionary } from '@onaio/utils';
+import { INVENTORY_SERVICE_POINT_LIST_VIEW } from '@opensrp/inventory';
+import {
+  ACTIVE_PLANS_LIST_VIEW_URL,
+  DRAFT_PLANS_LIST_VIEW_URL,
+  COMPLETE_PLANS_LIST_VIEW_URL,
+  TRASH_PLANS_LIST_VIEW_URL,
+} from '@opensrp/plans';
+import { CATALOGUE_LIST_VIEW_URL } from '@opensrp/product-catalogue';
+import ArchiveOutline from '@opensrp/ant-icons/lib/ArchiveOutline';
+import MapMarkerOutline from '@opensrp/ant-icons/lib/MapMarkerOutline';
+import { MenuItems } from '.';
+import {
+  ENABLE_PLANS,
+  ENABLE_INVENTORY,
+  ENABLE_LOCATIONS,
+  ENABLE_PRODUCT_CATALOGUE,
+  ENABLE_TEAMS,
+  ENABLE_FORM_CONFIGURATION,
+  ENABLE_CARD_SUPPORT,
+} from '../../../configs/env';
+import {
+  ACTIVE,
+  DRAFT,
+  COMPLETE,
+  TRASH,
+  SERVICE_POINT_INVENTORY,
+  USERS,
+  USER_MANAGEMENT,
+  LOCATION_UNIT,
+  LOCATION_UNIT_GROUP,
+  PRODUCT_CATALOGUE,
+  TEAMS,
+  FORM_CONFIGURATION,
+  MANIFEST_RELEASES,
+  DRAFT_FILES,
+  JSON_VALIDATORS,
+  MISSIONS,
+  ADMIN,
+  INVENTORY,
+} from '../../../lang';
+
+import {
+  URL_USER,
+  URL_LOCATION_UNIT,
+  URL_LOCATION_UNIT_GROUP,
+  URL_ADMIN,
+  URL_TEAMS,
+  URL_MANIFEST_RELEASE_LIST,
+  URL_DRAFT_FILE_LIST,
+  URL_JSON_VALIDATOR_LIST,
+  URL_DOWNLOAD_CLIENT_DATA,
+} from '../../../constants';
+
+// menu items schema
+export const menusSchema = (roles: Dictionary): MenuItems[] => [
+  {
+    otherProps: { icon: <MapMarkerOutline className="sidebar-icons" />, title: `${MISSIONS}` },
+    key: 'missions',
```

Comment:
This key must match the route?

Reply:
I do not think so, one of the benefits of this approach or at least as a I had imagined it is that, the keys itself is decoupled from the route, you can have a completeley random key and it would still work.

Reply:
@kelvin-muchiri the `url` prop is the one that is supposed to match the route